vertical-pod-autoscaler/pkg/target/fetcher.go

```diff
@@ -213,6 +213,9 @@ func (f *vpaTargetSelectorFetcher) getLabelSelectorFromResource(
 		groupResource := mapping.Resource.GroupResource()
 		scale, err := f.scaleNamespacer.Scales(namespace).Get(groupResource, name)
 		if err == nil {
+			if scale.Status.Selector == "" {
+				return nil, fmt.Errorf("Resource %s/%s has an empty selector for scale sub-resource", namespace, name)
```

Comment:
nit: please also log the resource type

Reply:
Actually, scratch that, the resource type is added to the error in the calling function